applications/tari_validator_node/tests/cucumber.rs

```diff
@@ -396,6 +546,33 @@ async fn wait_seconds(_world: &mut TariWorld, seconds: u64) {
     tokio::time::sleep(Duration::from_secs(seconds)).await;
 }
 
+#[then(expr = "the transaction succeeds")]
```

Comment:
If there are more than committee_size (7) VNs this won't work currently but this is fine for now
```suggestion
#[then(expr = "all transactions succeed on all validator nodes")]
```